src/en/news/blog/2008/blog/index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Is this post needed?

The content describes why the blog was setup.

Reply:
as the first blog post, i think so!